week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,88 @@
 // Write your code here
+let slides = [
+  "https://images.unsplash.com/photo-1470770841072-f978cf4d019e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80",
+  "https://images.unsplash.com/photo-1494500764479-0c8f2919a3d8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=600&q=60",
+  "https://images.unsplash.com/photo-1591727105417-a0a1055f48c9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=584&q=80",
+];
+
+const body = document.querySelector("body");
+body.style.backgroundColor = "pink";
+
+const divBackButton = document.querySelector(".backButton");
```

Comment:
At the moment you are using `querySelector` in order to access these elements however they're not in the html page at the moment. I think maybe you forgot to update the html page too ?